lib/fog/openstack/storage/requests/get_object_https_url.rb

```diff
@@ -54,14 +56,19 @@ def create_temp_url(container, object, expires, method, options = {})
           string_to_sign = "#{method}\n#{expires}\n#{object_path_unescaped}"
 
           hmac = Fog::HMAC.new('sha1', @openstack_temp_url_key.to_s)
-          sig  = sig_to_hex(hmac.sign(string_to_sign))
+
+          query = {
+            temp_url_sig: sig_to_hex(hmac.sign(string_to_sign)),
+            temp_url_expires: expires
+          }
+          query[:filename] = options[:filename] unless options[:filename].nil?
```

Comment:
Thanks for the PR! Just a nit, it might be easier to read ```query[:filename] = options[:filename] if options[:filename]```

Reply:
Sure, code updated. Thanks :)